lib/Jsrt/Jsrt.cpp

```diff
@@ -4293,11 +4293,28 @@ _ALWAYSINLINE JsErrorCode CompileRun(
 {
     PARAM_NOT_NULL(scriptVal);
     VALIDATE_JSREF(scriptVal);
+    PARAM_NOT_NULL(sourceUrl);
```

Comment:
>PARAM_NOT_NULL(sourceUrl); [](start = 4, length = 26)

Not sure if we have convention anywhere else or not -- do we want to enable passing nullptr(JsInvalidReference) to mean user does not care source url?